python/setup.py

```diff
@@ -21,6 +22,10 @@
     else:
         raise Exception('Could not find Eigen3 package!')
 
+    if __install_robot_model_module__:
+        __include_dirs__.append('/usr/local/include/osqp')
+        __include_dirs__.append('/opt/openrobots/include')
```

Comment:
If you want to support flexibility around these hard paths, I would suggest using environment variables along the lines of:

```python3
# get the path from an env variable if it's present
osqp_path = '/usr/local/include/osqp'
if 'OSQP_INCLUDE_DIR' in os.environ.keys():
  osqp_path = os.environ['OSQP_INCLUDE_DIR']

# or, same thing as a one-liner
osqp_path = os.environ['OSQP_INCLUDE_DIR'] if 'OSQP_INCLUDE_DIR' in os.environ.keys() else '/usr/local/include/osqp'

# then append
__include_dirs__.append(osqp_path)

# same with 'OPENROBOTS_INCLUDE_DIR'
```

When installing the python bindings you can then do the following
```shell
export OPENROBOTS_INCLUDE_DIR='/my/openrobots/path/include'
python -m pip install ./python
```

Preferably you would declare the env variable names and default paths up near the top of the file like `osqp_path_var = 'OSQP_INCLUDE_DIR'` so that it's easy to see and doesn't need to be duplicated (simply `os.environ[osqp_path_var]`)